src/hotspot/share/services/diagnosticCommand.cpp

```diff
@@ -1151,3 +1155,41 @@ void CompilationMemoryStatisticDCmd::execute(DCmdSource source, TRAPS) {
   const size_t minsize = _minsize.has_value() ? _minsize.value()._size : 0;
   CompilationMemoryStatistic::print_all_by_size(output(), human_readable, minsize);
 }
+
+SystemMapDCmd::SystemMapDCmd(outputStream* output, bool heap) :
```

Comment:
Shouldn't this all be inside `ifdef LINUX`? You don't need `os::realpath` if you do that.

Reply:
Well to call `os::Posix::realpath` it would need Linux specific includes ... but as it is you have unused code in the Windows version.